### PR TITLE
Feature/816 migration scripts

### PR DESCRIPTION
**Pull request by coltonglasgow**

### CODE REVIEW

File: migration/v12.0/migrate_endpoint.yaml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      password: "{password}"
+    queries:
+      source: "{reporting_queries_path_ngraphs}/"
+      includes:
+        - "detect_removed.rq"
+        - "detect_renamed.rq"
+        - "detect_old_namespace_subject.rq"
+        - "detect_old_namespace_predicate.rq"
+        - "detect_old_namespace_object.rq"
+        - "detect_old_namespace_graph.rq"
```
Comment (coltonglasgow) [inline migration/v12.0/migrate_endpoint.yaml:119]: I have been unable to test the validation against an endpoint. I am getting an error back that appears to be from the endpoint, I may have the wrong URL. I have tries several databases with no luck. @Jamie-SA, could you provide me with the endpoint URL for AGraph or GraphDB please.

**Review by Jamie-SA — CHANGES_REQUESTED** [review]
I noticed a few items and/or questions.

**Comment by Jamie-SA** [comment]
Colton is out for a couple of weeks so I'm going to work on addressing some of the issues I found during my review.

**Review by Jamie-SA — APPROVED** [review]
I fixed the issues I saw.

**Pull request merged**